packages/core/test-utils/src/utils.js

```diff
@@ -269,7 +270,7 @@ export function run(
   bundleGraph: BundleGraph<NamedBundle>,
   globals: mixed,
   opts: RunOpts = {},
-): Promise<any> {
```

Comment:
The only real "benefit" of `mixed` here is that you need to always add a `$FlowFixMe` when using it, so I'm not sure if this is better than before.

Reply:
Of the four tests that were affected, three were able to refine `mixed` enough to exercise them without any `$FlowFixMe`s